### ISSUE TITLE
Add second model for comparision

### ISSUE THREAD

**Issue by jayeshmahapatra** [jayeshmahapatra/ArcFace-Embedding-Visualization#5]
- Add a model with just softmax norm loss to the training loop
- Modify visualization code so that embeddings from boththe softmax norm loss and arcface loss models are visualized side by side
